en/04_Changelogs/5.0.0.md

```diff
@@ -4,8 +4,16 @@ title: 5.0.0 (unreleased)
 
 # 5.0.0 (unreleased)
 
+Silverstripe CMS 5.0.0 is the first major release since November 2017. It's also the first major release since we adopted a formal [Major Release Policy](/Project_governance/Major_release_policy).
+
+This release contains many breaking changes and dependency upgrades. We tried our best to minimise upgrades woes, but you should allocate additional time to account for regression testing.
+
+This changelogs provides an exhaustive list changes between Silverstripe CMS 4 and 5.
+
 ## Overview
 
+- [Change to commercially supported modules](#supported-modules)
+- [Fixed dependencies](#fixed-depdencies)
```

Comment:
I think I most over-ridden one of my own commit at some stage or something, because there's a bunch of things I committed directly from GitHub that aren't there anymore.